core/app.cpp

```diff
@@ -1170,9 +1185,9 @@ void init(int cmdline_argc, const char *const *cmdline_argv) {
     }
     return s;
   };
-  command_history_string = argv[0];
-  for (int n = 1; n < argc; ++n)
-    command_history_string += std::string(" ") + argv_quoted(argv[n]);
+  command_history_string = raw_arguments_list.front();
```

Comment:
Is the command history string here erroneously omitting the name of the command being executed? That's popped from the front of the list at [line 1148](https://github.com/MRtrix3/mrtrix3/pull/2911/files#diff-ea9657f1eb7d593a574dd53c8caaf0b7b7ae26233db0e199aad234a0ee849c18R1148).

Reply:
Yes, good catch.